py-polars/polars/internals/lazy_functions.py

```diff
@@ -380,13 +380,83 @@ def sum(column: pli.Series) -> int | float:
 
 def sum(column: str | list[pli.Expr | str] | pli.Series | pli.Expr) -> pli.Expr | Any:
     """
-    Get the sum value.
+    Sum values in a column/Series, or horizontally across list of columns/expressions.
 
+    `pl.sum(str)` is syntactic sugar for:
+
+    >>> pl.col(str).sum()  # doctest: +SKIP
+
+    `pl.sum(list)` is syntactic sugar for:
+
+    >>> pl.fold(pl.lit(0), lambda x, y: x + y, list).alias("sum")  # doctest: +SKIP
+
+    Parameters
+    ----------
     column
-        Column(s) to be used in aggregation. Will lead to different behavior based on
-        the input:
-        - Union[str, Series] -> aggregate the sum value of that column.
-        - List[Expr] -> aggregate the sum value horizontally.
+        Column(s) to be used in aggregation.
+        This can be:
+        - a column name, or Series -> aggregate the sum value of that column/Series.
```

Comment:
There needs to be a blank line before a list like this, to make it render properly in the docs:

```suggestion
        This can be:

                - a column name, or Series -> aggregate the sum value of that column/Series.
```

Reply:
I've made the change. How big a job is it to build the docs locally? If not too involved, it would help to see the impact of any changes and might save some GitHub cycles?

Reply:
It's not too bad. In the `py-polars/docs` folder, run `make html` (takes a few minutes). That will generate a `build/` folder with an `html/index.html`. You can open that in your preferred browser.

Reply:
Thanks, works fine.